pkg/nuclide/ocaml-base/lib/MerlinProcess.js

```diff
@@ -269,6 +270,18 @@ function getPathToMerlin(): string {
     && global.atom.config.get('nuclide.nuclide-ocaml.pathToMerlin') || 'ocamlmerlin';
 }
 
+/**
+ * @return The set of arguments to pass to ocamlmerlin.
+ */
+function getMerlinFlags(): Array<String> {
+  const configVal = global.atom
+    && global.atom.config.get('nuclide.nuclide-ocaml.merlinFlags');
+  // To split while stripping out any leading/trailing space, we match on all
+  // *non*-whitespace.
+  const configItems = configVal && configVal.match(/\S+/g);
```

Comment:
We have `shell-quote` already and the Clang package uses it for this exact purpose. But I don't feel that strongly about it. :+1: